exporter/http.go

```
@@ -88,3 +89,19 @@ func (e *Exporter) scrapeHandler(w http.ResponseWriter, r *http.Request) {
 		registry, promhttp.HandlerOpts{ErrorHandling: promhttp.ContinueOnError},
 	).ServeHTTP(w, r)
 }
+
+func (e *Exporter) ReloadPwdFile(w http.ResponseWriter, r *http.Request) {
+	passwordMap := make(map[string]string, 8)
+	if e.options.RedisPwdFile != "" {
```

Should we return an error if pwd file is not set? E.g. a 400?